webapp/commonComponents/hooks/index.js

```diff
@@ -17,5 +17,5 @@ export { default as useProfilePicture } from './useProfilePicture'
 export { default as useOnIntersect } from './useOnIntersect'
 export { default as useInterval } from './useInterval'
 export { useSurvey, useSurveyId, useSurveyInfo, useSurveyCycleKey, useOnSurveyCycleUpdate } from './useSurvey'
-export { default as useNodeDef } from './useNodeDef'
+export { useNodeDef, useNodeDefValidation, useNodeDefIsDirty } from './useNodeDef'
```

Comment:
I'm thinking that these hooks aren't generic, used only in nodeDef view? or not?

Reply:
if that's the case, you can either remove them or move in the nodeDef folder

Reply:
uhm, you are right